.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'type: bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Link to microCOVID scenario**
+If applicable, please copy-paste the URL to the scenario that you modeled on the calculator, so we can easily reproduce it.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Additional context / screenshots**
+Add any other context about the problem here, including any screenshots.
```

Comment:
ooh, maybe add 

**Is this a regression?**
Was this feature working properly before? If so, when was the last time it worked the way you expected?